generated/nidaqmx/constants.py

```diff
@@ -907,10 +931,10 @@ class UnitsPreScaled(Enum):
     DEG_F = 10144  #: Degrees Fahrenheit.
     DEG_C = 10143  #: Degrees Celsius.
     DEG_R = 10145  #: Degrees Rankine.
-    K = 10325  #: Kelvins.
+    KELVINS = 10325  #: Kelvins.
```

Comment:
whatever you do on the previous instance, match it here

Reply:
Updated enum values as in expected constants.py